IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -1679,6 +1679,7 @@ public enum ImageFormat
 
         }
 
+#pragma warning disable CS0618
```

Comment:
IMO, best to mark this method obsolete too, rather than disable a warning. The point of Obsolete is to remind devs to stop using methods, because these methods were made obsolete for a reason (unsafe, bad performance, about to be removed, .etc). So whoever will be using this hack needs to be also reminded, that they need to rethink their solution.

Reply:
Fair point. Changing it in the other PR.